dashboard/config/videos.csv

```diff
@@ -104,11 +104,13 @@ tutorial_puzzle_challenge			_l68chtl1co	http://videos.code.org/2014/tutorial-puz
 infinity_playlab_intro			vQGOfhsb76o	http://videos.code.org/2015/infinityplaylab/infinity-playlab-intro.mp4
 infinity_playlab_events			HcljJx_0Elw	http://videos.code.org/2015/infinityplaylab/infinity-playlab-events.mp4
 infinity_playlab_repeat_forever			Ldj1-kEa9nA	http://videos.code.org/2015/infinityplaylab/infinity-playlab-repeat-forever.mp4
+csd_gamelab_animation_tab			DQPruYTJ0Fk	https://videos.code.org/cs-discoveries/gamelab/gamelab_animation_tab.mp4
 csd_gamelab_draw_1			PXn9gKiKKFo	https://videos.code.org/cs-discoveries/gamelab/gamelab_drawing_1.mp4
 csd_gamelab_draw_2			QLabeWhw_O0	https://videos.code.org/cs-discoveries/gamelab/gamelab_drawing_2.mp4
 csd_gamelab_drawloop			G6QJeuHhqCM	https://videos.code.org/cs-discoveries/gamelab/gamelab_drawloop.mp4
 csd_gamelab_sprites_1			aUHzpGt9nso	https://videos.code.org/cs-discoveries/gamelab/gamelab_sprites.mp4
 csd_gamelab_sprites_2			G666sFzAg5g	https://videos.code.org/cs-discoveries/gamelab/gamelab_animating_sprites.mp4
+csd_gamelab_velocity			zcP94yMYqrI	https://videos.code.org/cs-discoveries/gamelab/gamelab_velocity.mp4
```

Comment:
We've had some issues with thumbnails when adding videos to the middle of the list in the past. It would probably be safer to append new videos to the end of the list.